lib/WebpackConfig.js

```diff
@@ -405,10 +405,6 @@ class WebpackConfig {
                 normalizedOptionKey = 'includeNodeModules';
             }
 
-            if (['useBuiltIns', 'corejs'].includes(optionKey)) {
```

Comment:
> If we don't deprecate the options, it's possible to do:
> ```js
> Encore
>  .configureBabel(null, {
>    corejs: 3,
>  })
>  .configureBabelPresetEnv((options) => {
>    options.corejs: 2
>  });

I don't think that's a real issue, the options have the same name and it's quite unlikely someone will try to set them in both `configureBabel()` and `configureBabelPresetEnv()` (and even more unlikely that they will with different values).

That's also the case for other methods in Encore, for instance if you call `Encore.enableVueLoader(() => {}, { useJsx: true })`, and `Encore.configureBabel(options => { options.presets = []; })`.

> I think it's better to update the flex recipe instead of not deprecating the options because the recipe recommends them.

It doesn't really cost us anything to keep them here, but it will cost time to users that initialized Encore using the flex recipe if we remove them.

But as I said that's not the only reason: I think having some real options that we can document / add checks on beside callbacks is a good thing. Maybe it could be argued that in this case they should be set by `configureBabelPresetEnv()` but that's not a big issue either (it's still related to Babel after all).